src/bin/parser.js

```diff
@@ -439,33 +440,33 @@ export default class Harmonic {
         return pages;
     }
 
-    getPostFiles() {
+    async getPostFiles() {
         const files = {};
 
         for (const lang of this.config.i18n.languages) {
-            files[lang] = fs.readdirSync(path.join(this.sitePath, postspath, lang))
-                .filter((p) => rMarkdownExt.test(p));
+            files[lang] = await fs.readdirAsync(path.join(this.sitePath, postspath, lang));
+            files[lang] = files[lang].filter((p) => rMarkdownExt.test(p));
         }
 
         return files;
     }
 
-    getPageFiles() {
+    async getPageFiles() {
         const files = {};
 
         for (const lang of this.config.i18n.languages) {
             const langPath = path.join(this.sitePath, pagespath, lang);
-            mkdirp.sync(langPath);
-            files[lang] = fs.readdirSync(langPath).filter((p) => rMarkdownExt.test(p));
+            await mkdirpAsync(langPath);
+            files[lang] = await fs.readdirAsync(langPath).filter((p) => rMarkdownExt.test(p));
```

Comment:
As (AFAIK) the precedence of the `await` operator is not well defined yet, I'd recommend parenthesising await expressions.